Voter/AclVoter.php

```diff
@@ -26,8 +26,10 @@
  * This voter can be used as a base class for implementing your own permissions.
  *
  * @author Johannes M. Schmitt <schmittjoh@gmail.com>
+ *
+ * @internal
  */
-class AclVoter implements VoterInterface
+abstract class InternalAclVoter implements VoterInterface
```

Comment:
I'd rather make this a trait, that would also prevent the indirection via any protected method (aka we can remove internalVote method)

Reply:
@nicolas-grekas Please review if that is what you expected